src/shogun/preprocessor/RandomFourierGaussPreproc.cpp

```diff
@@ -14,120 +14,103 @@
 
 using namespace shogun;
 
+//TODO: Haven't handled this 
```

Comment:
no need for such comments, pls don't commit them into the PR. You can do them locally still, but no need to pollute the diff here